src/Views/Form/Themes/Sequoia/Actions.php

```diff
@@ -132,13 +135,43 @@ public function getProgressBarSection( $form_id, $args, $form ) {
 	 *
 	 * @since 2.7.0
 	 */
-	public function getNextButton() {
+	public function getNextButton( $id ) {
+
+		// Get Theme options
+		$theme_options = give_get_meta( $id, '_give_sequoia_form_theme_settings', true, null );
+
+		$label = isset( $theme_options['introduction']['next_label'] ) ? $theme_options['introduction']['next_label'] : __( 'Next', 'give' );
+		$color = isset( $theme_options['introduction']['primary_color'] ) ? $theme_options['introduction']['primary_color'] : '#2bc253';
+
 		printf(
-			'<div class="give-show-form give-showing__introduction-section"><button class="give-btn">%1$s</button></div>',
-			__( 'Next', 'give' )
+			'<div class="give-show-form give-showing__introduction-section"><button class="give-btn" style="background: %1$s">%2$s</button></div>',
+			$color,
+			$label
 		);
 	}
 
+	/**
+	 * Add checkout button
+	 *
+	 * @since 2.7.0
+	 */
+	public function getCheckoutButton() {
+		$session = give_get_purchase_session();
+		$payment = new \Give_Payment( $session['donation_id'] );
+
+		// Get Theme options
+		$theme_options = give_get_meta( $payment->form_id, '_give_sequoia_form_theme_settings', true, null );
```

Comment:
@henryholtgeerts We are querying for form theme setting multiple times in same class, can we call them once and reuse.

Reply:
Had the same thought, but wasn't sure how it will work with the receipt template. Regardless, I've implemented this concept now. Ref: https://github.com/impress-org/givewp/commit/b1375fdefbcc335bbbd76e159356f4008de5fcc4

Reply:
@henryholtgeerts this functionality is also blocker for me. I will review how we can get form setting and form shortcode arguments in an organized way in Theme.